spec/compiler/codegen/c_abi/c_abi_spec.cr

```diff
@@ -84,6 +84,36 @@ describe "Code gen: C ABI" do
       ), &.to_i.should eq(6))
   end
 
+  {% if flag?(:x86_64) && !flag?(:win32) %}
+    pending "passes struct after many other args (for real) (#9519)"
```

Comment:
Why the spec does not apply on non-windows platform?

Reply:
Because it's broken there too, but a bit harder to fix (and maybe arguably out of scope here, wanting to fix it for the windows ABI).

Reply:
Ok